<launch>
    <!-- This node will launch frame conversion from vrpn pose (geometry_msgs/PoseStamped) to mavros pose -->

    <param name="output_rate" value="30" />
    <!-- put other parameters here, e.g., tracker name -->
    <!-- <param name="tracker_name" value="Q2s" /> --> 

    <node pkg="vision_to_mavros" type="vicon_to_mavros_my_node" name="vicon_to_mavros" output="screen" >
        <remap from="vicon_pose" to="/mavros/vision_pose/pose" />
    </node>
</launch>
